.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,46 @@
+{
+	"name": "Nixpacks",
+	"dockerFile": "Dockerfile",
+	"runArgs": ["--init"],
+	"mounts": ["source=/var/run/docker.sock,target=/var/run/docker-host.sock,type=bind"],
+	"overrideCommand": false,
+	
+	// Use this environment variable if you need to bind mount your local source code into a new container.
+	"remoteEnv": {
+		"LOCAL_WORKSPACE_FOLDER": "${localWorkspaceFolder}"
+	},
+
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {	
+            "settings": { 
+				"lldb.executable": "/usr/bin/lldb",
+				// VS Code don't watch files under ./target
+				"files.watcherExclude": {
+					"**/target/**": true
+				},
+				"rust-analyzer.checkOnSave.command": "clippy"
+			},
+
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+				"ms-azuretools.vscode-docker",
+                "vadimcn.vscode-lldb",
+				"mutantdino.resourcemonitor",
+				"rust-lang.rust-analyzer",
+				"tamasfe.even-better-toml",
+				"serayuzgur.crates"
```

Comment:
personally I don't like using this extension, is there a way to disable it from being installed just for me? I looked around the docs and couldn't find it.

Reply:
Which one?

Reply:
The crates one

Reply:
Alright, will remove it. (it is annoying for me too)

Reply:
Removed in https://github.com/devtomio/nixpacks/commit/e328915f7fe49828712b224f06c35d0ace58cd1c